WTWSMS/events/religious_events.txt

```diff
@@ -772,7 +790,7 @@ character_event = {
 			is_heretic = no
 			prisoner = no
 			NOT = { trait = incapable }
-			reverse_opinion = { who = ROOT value = 25 }
+			reverse_opinion = { who = ROOT value = 10 }
```

Comment:
Normally, you don't explain to me why my values are too low or too high. I'll do the same here.